content/flare/consensus.md

```diff
@@ -287,3 +287,223 @@ Market forces cause quorum slices to overlap and this gives rise to the network-
+
+A process has no control over which message extends its casual order prefix in a step, and it cannot determine whether any future extension of its casual order prefix will involve a message that follows a particular message.
+The asynchronous nature and faulty behavior of the process and the communication medium are reflected in the Byzantine partial order of messages that is input to the algorithms.
+
+### Aleph Algorithm
+
+One of the examples of Total Order algorithms with total ordering execution is [Aleph](https://arxiv.org/abs/1908.05156).
+Aleph is a leaderless, fully asynchronous, Byzantine Fault Tolerant consensus protocol for total ordering of messages that are exchanged between entities in a network.
+Consequently, it is based on a distributed construction of a partially ordered set and the execution step - the algorithm for reaching a consensus on its extension to a total order.
+To achieve a consensus, the processes perform computations based on only a local copy of the data structure, however, they are bound to end up with the same result as required by the total ordering process.
+
+The nodes in Aleph protocol network send arbitrary messages to each other using a combination of multicast and random gossip for communication.
+Each node knows the identity of all other nodes and identifies itself through its public key, for which it holds a private key.
+
+From the high level perspective Aleph consensus algorithm functions on a set of nodes that listen for transactions.
+The nodes need to agree on a total ordering of the received transactions arriving in some [unreliable streams of data that the nodes observe locally](#Characteristics).
+The set of transactions that each one sees is a growing blockchain that does not have a built-in notion of finality and the streams of data are an advancing sequence of blockchain "tips" that each of the participants sees.
+
+The problem of ordering transactions in asynchronous types of protocols, where they are received with some time delay and in mixed order, is also known as _Atomic Broadcast_.
+It describes the situation where a set of nodes commonly constructs a total ordering of a set of transactions, where these transactions might not arrive all at the same time.
+Every time a node receives a transactions, it processes it and places it at its position.
+The sequence of transactions must be output in an incremental manner, meaning that before putting a transaction at some position, previous positions must have their respective transactions set.
+It is assumed that whenever an honest node outputs a transaction at some position, every other honest node will also eventually output the same transaction for this position.
+Also, every transaction that is input at some honest node is eventually output by all honest nodes.
+This ensures that not only all honest nodes will produce the same ordering, but also that no transactions is lost due to censorship because all honest nodes will output it.
```

Comment:
```suggestion
This ensures that not only all honest nodes produce the same ordering, but also that no transaction is lost due to censorship because all honest nodes output it.
```

Reply:
Corrected here https://github.com/optakt/optakt.github.io/pull/25/commits/e51539edef429ee0583f05806acf8610dd0971a0#diff-af4a1f964639da79798e8b30e7b99b5fb3f29ea0d4da47acd88f0d9c9867be13R400
Thanks!